Building Fantomas in Visual Studio for dev errors

I'm getting an error while trying to build this project:

```
1>------ Rebuild All started: Project: Fantomas.FCS, Configuration: Debug Any CPU ------
Restored X:\dev\forks\fantomas\src\Fantomas.Core.Tests\Fantomas.Core.Tests.fsproj (in 150 ms).
Restored X:\dev\forks\fantomas\src\Fantomas.Benchmarks\Fantomas.Benchmarks.fsproj (in 160 ms).
Restored X:\dev\forks\fantomas\src\Fantomas.FCS\Fantomas.FCS.fsproj (in 53 ms).
Restored X:\dev\forks\fantomas\src\Fantomas\Fantomas.fsproj (in 241 ms).
Restored X:\dev\forks\fantomas\src\Fantomas.Client\Fantomas.Client.fsproj (in 243 ms).
Restored X:\dev\forks\fantomas\src\Fantomas.Tests\Fantomas.Tests.fsproj (in 254 ms).
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: The "Ionide.KeepAChangelog.Tasks.ParseChangeLogs" task failed unexpectedly.
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: System.IO.FileNotFoundException: Could not load file or assembly 'FSharp.Core, Version=4.4.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: File name: 'FSharp.Core, Version=4.4.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018:    at Ionide.KeepAChangelog.Parser.parseChangeLog(FileInfo file)
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018:    at KeepAChangelog.Tasks.ParseChangelogs.Execute() in /home/runner/work/KeepAChangelog/KeepAChangelog/src/Ionide.KeepAChangelog.Tasks/Library.fs:line 117
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: 
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: WRN: Assembly binding logging is turned OFF.
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: Note: There is some performance penalty associated with assembly bind failure logging.
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018: To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].
1>C:\Users\xyz\.nuget\packages\ionide.keepachangelog.tasks\0.1.8\build\Ionide.KeepAChangelog.Tasks.targets(29,9): error MSB4018:
```

I suspect it to be related to this https://github.com/ionide/KeepAChangelog/issues/8

I'm able to workaround the issue locally by commenting out `Ionide.KeepAChangelog` in a couple of paket.references files then rebuilding. I'm not sure if there's a better workaround.

I do not get the error when building on the command line with FAKE.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+### Documented
+* Building Fantomas in Visual Studio for dev errors. [#2447](https://github.com/fsprojects/fantomas/issues/2447)
+
 ## [5.0.1] - 2022-09-18
 
 ### Fixed

--- a/docs/docs/contributors/Index.md
+++ b/docs/docs/contributors/Index.md
@@ -21,4 +21,10 @@ The best is yet to come!
 PS: Don't hesitate to open [an issue](https://github.com/fsprojects/fantomas/issues/new/choose) if you have any questions. 
 Or if something isn't all that clear. Our goal is to make this documentation as complete as possible🎉!
 
+## Visual Studio Issue
+
+There is currently [a known issue](https://github.com/fsprojects/fantomas/issues/2447) when loading the `fantomas` solution in Visual Studio that prevents the solution from being able to build properly. This is due to an issue in the `Ionide.KeepAChangelog` build step (tracking the issue [here](https://github.com/ionide/KeepAChangelog/issues/8)).
+
+To workaround this in the meantime, you'll need to comment out the references to `Ionide.KeepAChangelog.Tasks` in `paket.dependencies` and all of the `paket.references` files, and then run `dotnet paket install` again. 
+
 <fantomas-nav next="./FSharp.html"></fantomas-nav>

--- a/docs/docs/contributors/Index.md
+++ b/docs/docs/contributors/Index.md
@@ -25,6 +25,6 @@ Or if something isn't all that clear. Our goal is to make this documentation as 
 
 There is currently [a known issue](https://github.com/fsprojects/fantomas/issues/2447) when loading the `fantomas` solution in Visual Studio that prevents the solution from being able to build properly. This is due to an issue in the `Ionide.KeepAChangelog` build step (tracking the issue [here](https://github.com/ionide/KeepAChangelog/issues/8)).
 
-To workaround this in the meantime, you'll need to comment out the references to `Ionide.KeepAChangelog.Tasks` in `paket.dependencies` and all of the `paket.references` files, and then run `dotnet paket install` again. 
+To workaround this in the meantime, you'll need to comment out the references to `Ionide.KeepAChangelog.Tasks` in `paket.dependencies` and all of the `paket.references` files, and then run `dotnet paket install` again. **Please be careful not to include these changes when submitting a PR!**
 
 <fantomas-nav next="./FSharp.html"></fantomas-nav>

